app/app.js

```diff
@@ -2,16 +2,18 @@ const express = require('express')
 const app = express()
 const bodyParser = require('body-parser')
 const nunjucks = require('nunjucks')
-const util = require('util')
-const fs = require('fs')
 const path = require('path')
 
-const readdir = util.promisify(fs.readdir)
-
 const helperFunctions = require('../lib/helper-functions')
-const fileHelper = require('../lib/file-helper')
 const configPaths = require('../config/paths.json')
 
+// Routers
+const indexRouter = require('./routes/indexRoutes')
```

Comment:
🤔 At first I thought it would nice to be clear what routers we were using at the top by declaring them altogether. It was also difficult at first because there was a lot of router details in and amongst this file.

Looking at the current state I think we should just inline the requires to the routers as you suggest because all the routes are now being called relatively close together and we do not use those variables anywhere else.

👍